### PR TITLE
Add API css build

### PR DESCRIPTION
**Pull request by tonio**
and a makefile recipe to watch & rebuild.

### CODE REVIEW

**Comment by tonio** [comment]
Clean & lint targets added.

**Comment by tonio** [comment]
Route to test loader added. Ready for review.

File: geoportailv3.mk [hunk 1]
```diff
@@ -114,3 +117,16 @@ $(API_OUTPUT_DIR)/apiv3.js: $(API_DIR)/config.json \
...
+lint-api: ./node_modules/.bin/eslint .build/node_modules.timestamp .build/api.eslint.timestamp
+
+.build/api.eslint.timestamp: $(API_JS_FILES)
+	mkdir -p $(dir $@)
+	./node_modules/.bin/eslint $(filter-out .build/node_modules.timestamp, $?)
+	touch $@
+
+clean: geoportailv3clean
+
+geoportailv3clean:
```
Comment (pgiraud) [inline geoportailv3.mk:131]: Shouldn't this be called clean-api?

Comment (tonio) [inline geoportailv3.mk:131]: Indeed.

**Pull request merged**